src/dynamics/rigid_body_components.rs

```diff
@@ -972,7 +972,7 @@ impl RigidBodyColliders {
 
             // Set the modification flag so we can benefit from the modification-tracking
             // when updating the narrow-phase/broad-phase afterwards.
-            co.changes |= ColliderChanges::POSITION;
+            co.changes |= ColliderChanges::MODIFIED | ColliderChanges::POSITION;
```

Comment:
This makes me wonder if we should change `ColliderChanges::POSITION` to  `const POSITION = 1 << 3 | 1 << 0;` (adding `ColliderChanges::MODIFIED` to it), this would clarify their relation and make mistakes harder to make.

Reply:
This change is probably not possible because `MODIFIED` is used to check if a collider is part of the `modified_colliders` list. This seems difficult to maintain, I imagine searching for `modified_colliders.push`  should be able to lead to a `MODIFIED` bit to be added to that collider, but it's sometimes tricky to assess. 

I imagine changing `modified_colliders` to a `Hashmap` may be less performant than storing that in the bitfield, but I'm throwing the idea.